modules/ROOT/pages/changelogs.adoc

```diff
@@ -22,14 +35,31 @@ For more information, see link:https://neo4j.com/docs/operations-manual/current/
 
 **New:**
 
-[source, status codes, role="noheader"]
------
-Neo.ClientNotification.Statement.RedundantOptionalProcedure
-Neo.ClientNotification.Statement.RedundantOptionalSubquery
-Neo.ClientNotification.Security.AuthProviderNotDefined
-Neo.ClientNotification.Security.ExternalAuthNotEnabled
-Neo.ClientNotification.Statement.AggregationSkippedNull
------
+[options="header", cols="<1m,<1"]
+|===
+
+| GQLSTATUS
+| Neo4j code
+
+| 00N72
+| Neo.ClientNotification.Security.AuthProviderNotDefined
+
+| 01G11
+| Neo.ClientNotification.Statement.AggregationSkippedNull
+
+| 01N71
+| Neo.ClientNotification.Security.ExternalAuthNotEnabled
+
+| 03N61
+| Neo.ClientNotification.Statement.RedundantOptionalProcedure
+
+| 03N62
+| Neo.ClientNotification.Statement.RedundantOptionalSubquery
+
+| ?? // Not assigned yet and not documented
```

Comment:
This one is not documented, needs to be added to the "List of Neo4j error codes" page also. 

Also it is not decided which GQLSTATUS code(s) it will correspond to, not sure how we want to mark that